tests/commands/test_version_command.py

```diff
@@ -70,3 +74,30 @@ def test_version_for_showing_commitizen_system_info(config, capsys):
     assert f"Commitizen Version: {__version__}" in captured.out
     assert f"Python Version: {sys.version}" in captured.out
     assert f"Operating System: {platform.system()}" in captured.out
+
+
+@pytest.mark.parametrize("project", (True, False))
+@pytest.mark.usefixtures("tmp_git_project")
+def test_version_use_version_provider(
+    mocker: MockerFixture,
+    config: BaseConfig,
+    capsys: pytest.CaptureFixture,
+    project: bool,
+):
+    version = "0.0.0"
+    mock = mocker.MagicMock(name="provider")
+    mock.get_version.return_value = version
+    get_provider = mocker.patch(
+        "commitizen.commands.version.get_provider", return_value=mock
+    )
+
+    commands.Version(
+        config,
+        {"report": False, "project": project, "commitizen": False, "verbose": True},
```

Comment:
I'd suggest we changing the test case here. These 4 are in the same [exclusive_group](https://github.com/commitizen-tools/commitizen/blob/master/commitizen/cli.py#L315) and thus only one of them can be `True`. Although it does not affect the functionality, it'd still be better if we could keep the test cases closer to our use cases. Something like `@pytest.mark.parametrize("project, verbose", ((True, False), (False, True)))` would work. Thanks!

Reply:
Solved with `verbose = not project`